2023/03/17 16:23:32 [error] 17#17: *207340 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:32 [error] 17#17: *213317 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:32 [error] 17#17: *213321 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:32 [error] 16#16: *213322 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:32 [error] 17#17: *144761 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:32 [error] 16#16: *213329 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:32 [error] 16#16: *213331 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:32 [error] 17#17: *182380 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:32 [error] 18#18: *197279 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:32 [error] 16#16: *213334 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:32 [error] 17#17: *213338 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:32 [error] 17#17: *213337 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
2023/03/17 16:23:32 [error] 17#17: *196109 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:32 [error] 17#17: *210577 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:32 [error] 18#18: *194303 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:32 [error] 16#16: *213340 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:32 [error] 17#17: *207388 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:32 [error] 16#16: *213348 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:32 [error] 16#16: *213350 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:32 [error] 19#19: *207384 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:32 [error] 17#17: *202848 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:32 [error] 17#17: *213339 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:32 [error] 17#17: *191520 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:32 [error] 17#17: *207392 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:32 [error] 18#18: *196029 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:32 [error] 18#18: *193941 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:32 [error] 16#16: *213356 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:32 [error] 16#16: *213366 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:32 [error] 17#17: *205389 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:32 [error] 17#17: *207443 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:32 [error] 18#18: *208777 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:32 [error] 18#18: *196587 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:32 [error] 16#16: *213372 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:32 [error] 18#18: *210650 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:32 [error] 18#18: *207330 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:32 [error] 19#19: *207644 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:32 [error] 18#18: *174038 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:32 [error] 16#16: *213378 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:32 [error] 17#17: *209846 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:32 [error] 19#19: *194796 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:32 [error] 17#17: *122576 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:32 [error] 16#16: *214892 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST /api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST /api/user/login HTTP/1.1" 502 494 "-" "Apache-HttpClient/4.5.12 (Java/19.0.1)" "-"
2023/03/17 16:23:32 [error] 16#16: *213387 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:32 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:33 [error] 17#17: *207389 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 18#18: *179300 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:33 [error] 18#18: *192842 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:33 [error] 16#16: *213391 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:33 [error] 18#18: *184052 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 16#16: *213398 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:33 [error] 16#16: *213401 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 17#17: *210736 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:33 [error] 17#17: *207706 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:33 [error] 17#17: *207488 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:33 [error] 17#17: *213392 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:33 [error] 16#16: *213410 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:33 [error] 17#17: *213411 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:33 [error] 17#17: *213397 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 18#18: *186806 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:33 [error] 19#19: *207640 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 19#19: *195832 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET / HTTP/1.1" 200 2610 "-" "Apache-HttpClient/4.5.12 (Java/19.0.1)" "-"
2023/03/17 16:23:33 [error] 16#16: *213419 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:33 [error] 17#17: *207723 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:33 [error] 19#19: *210214 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:33 [error] 17#17: *178747 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:33 [error] 17#17: *193300 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:33 [error] 17#17: *213409 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:33 [error] 16#16: *213429 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
2023/03/17 16:23:33 [error] 17#17: *213420 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:33 [error] 17#17: *213428 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 17#17: *176696 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:33 [error] 19#19: *121462 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:33 [error] 16#16: *213434 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:33 [error] 16#16: *213436 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:33 [error] 17#17: *189180 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:33 [error] 18#18: *205048 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:33 [error] 16#16: *213447 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:33 [error] 17#17: *200097 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 19#19: *209898 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:33 [error] 17#17: *213440 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:33 [error] 17#17: *191894 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:33 [error] 16#16: *213457 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 17#17: *210090 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:33 [error] 16#16: *213460 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
2023/03/17 16:23:33 [error] 19#19: *207638 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:33 [error] 18#18: *209869 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:33 [error] 16#16: *213469 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 16#16: *213472 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:33 [error] 17#17: *180364 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:33 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:34 [error] 17#17: *208316 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:34 [error] 16#16: *213475 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:34 [error] 17#17: *171670 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:34 [error] 17#17: *181343 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:34 [error] 19#19: *196429 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:34 [error] 17#17: *191606 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:34 [error] 16#16: *213483 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:34 [error] 16#16: *213485 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:34 [error] 18#18: *210494 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:34 [error] 19#19: *184123 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:34 [error] 16#16: *213487 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:34 [error] 16#16: *213491 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:34 [error] 18#18: *194316 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:34 [error] 17#17: *207859 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:34 [error] 17#17: *210911 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:34 [error] 17#17: *195507 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:34 [error] 16#16: *213496 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:34 [error] 16#16: *213498 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:34 [error] 17#17: *207598 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:34 [error] 17#17: *213502 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:34 [error] 16#16: *213501 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:34 [error] 16#16: *213505 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:34 [error] 17#17: *209912 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:34 [error] 19#19: *197504 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:34 [error] 17#17: *191929 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:34 [error] 16#16: *213510 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:34 [error] 16#16: *213512 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:34 [error] 17#17: *205422 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:34 [error] 18#18: *210547 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:34 [error] 17#17: *185089 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:34 [error] 17#17: *210722 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:34 [error] 16#16: *213518 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:34 [error] 17#17: *186805 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:34 [error] 17#17: *208163 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
2023/03/17 16:23:34 [error] 16#16: *213525 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:34 [error] 17#17: *205192 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:34 [error] 17#17: *207807 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:34 [error] 16#16: *213529 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:34 [error] 17#17: *213535 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
2023/03/17 16:23:34 [error] 19#19: *210742 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:34 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:34 [error] 16#16: *213532 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:35 [error] 16#16: *213534 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:35 [error] 16#16: *213538 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:35 [error] 16#16: *213541 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:35 [error] 16#16: *213543 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:35 [error] 16#16: *213545 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:35 [error] 16#16: *213547 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:35 [error] 17#17: *205047 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:35 [error] 16#16: *213550 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:35 [error] 17#17: *207860 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:35 [error] 19#19: *195831 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
2023/03/17 16:23:35 [error] 16#16: *213558 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
2023/03/17 16:23:35 [error] 18#18: *180363 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:35 [error] 16#16: *213561 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:35 [error] 17#17: *196548 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:35 [error] 18#18: *197165 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:35 [error] 17#17: *207902 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:35 [error] 16#16: *213565 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:35 [error] 16#16: *213570 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:35 [error] 17#17: *202845 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:35 [error] 17#17: *179106 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:35 [error] 17#17: *210475 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:35 [error] 17#17: *210844 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:35 [error] 17#17: *207597 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:35 [error] 17#17: *217694 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:35 [error] 19#19: *207688 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:35 [error] 17#17: *213578 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:35 [error] 19#19: *189533 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:35 [error] 18#18: *210497 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:35 [error] 17#17: *207407 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:35 [error] 17#17: *210544 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:35 [error] 18#18: *213585 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:35 [error] 19#19: *210548 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:35 [error] 17#17: *202900 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:35 [error] 17#17: *125116 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:35 [error] 17#17: *184317 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:35 [error] 18#18: *210647 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:35 [error] 17#17: *210460 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:35 [error] 19#19: *210648 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:35 [error] 17#17: *195593 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:35 [error] 17#17: *192638 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:35 [error] 18#18: *213598 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:35 [error] 18#18: *213597 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:35 [error] 18#18: *213596 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:35 [error] 17#17: *194501 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:35 [error] 18#18: *196730 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:35 [error] 17#17: *208015 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:35 [error] 17#17: *213605 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:35 [error] 17#17: *213606 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:35 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:36 [error] 17#17: *184415 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:36 [error] 17#17: *177267 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:36 [error] 17#17: *177685 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:36 [error] 17#17: *213612 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:36 [error] 17#17: *188300 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:36 [error] 17#17: *213615 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:36 [error] 17#17: *208339 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:36 [error] 17#17: *210620 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:36 [error] 19#19: *213918 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:36 [error] 18#18: *213621 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:36 [error] 17#17: *213620 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:36 [error] 18#18: *197139 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:36 [error] 17#17: *202902 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:36 [error] 17#17: *213626 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:36 [error] 17#17: *194827 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:36 [error] 18#18: *121723 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:36 [error] 17#17: *206017 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:36 [error] 19#19: *205052 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:36 [error] 18#18: *196230 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:36 [error] 18#18: *210980 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:36 [error] 19#19: *196600 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:36 [error] 16#16: *216742 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST /api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST /api/user/login HTTP/1.1" 502 494 "-" "Apache-HttpClient/4.5.12 (Java/19.0.1)" "-"
2023/03/17 16:23:36 [error] 18#18: *198795 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:36 [error] 17#17: *198726 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:36 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:37 [error] 18#18: *210492 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
2023/03/17 16:23:37 [error] 17#17: *213639 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
2023/03/17 16:23:37 [error] 17#17: *213640 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:37 [error] 16#16: *213643 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST /api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "POST /api/user/login HTTP/1.1" 504 494 "-" "Apache-HttpClient/4.5.12 (Java/19.0.1)" "-"
2023/03/17 16:23:37 [error] 18#18: *196425 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:37 [error] 18#18: *197501 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:37 [error] 19#19: *205139 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "POST //api/user/login HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:37 [error] 19#19: *194797 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET //api/user/uniqueid HTTP/1.1" 504 494 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET / HTTP/1.1" 200 2610 "-" "Apache-HttpClient/4.5.12 (Java/19.0.1)" "-"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET / HTTP/1.1" 200 2610 "-" "Apache-HttpClient/4.5.12 (Java/19.0.1)" "-"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:37 [error] 16#16: *216742 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST /api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "POST /api/user/login HTTP/1.1" 502 494 "-" "Apache-HttpClient/4.5.12 (Java/19.0.1)" "-"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:37 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET / HTTP/1.1" 200 2610 "-" "Apache-HttpClient/4.5.12 (Java/19.0.1)" "-"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:38 [error] 19#19: *207453 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:38 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:39 [error] 19#19: *199956 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:39 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:40 [error] 17#17: *212437 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:40 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:41 [error] 16#16: *212852 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:41 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:42 [error] 17#17: *213745 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:42 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:43 [error] 16#16: *212966 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:43 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:44 [error] 19#19: *196896 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:44 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:45 [error] 16#16: *215644 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:45 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:46 [error] 18#18: *196949 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:46 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:47 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:47 [error] 17#17: *194827 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:47 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:48 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:48 [error] 19#19: *217155 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:48 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:49 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:49 [error] 18#18: *189519 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:49 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:50 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:50 [error] 16#16: *219291 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:50 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:51 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
2023/03/17 16:23:51 [warn] 17#17: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:51 [error] 16#16: *219466 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:51 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:52 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
2023/03/17 16:23:52 [warn] 17#17: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:52 [error] 16#16: *219465 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:52 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
2023/03/17 16:23:53 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:53 [warn] 17#17: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "192.241.230.151"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:53 [error] 16#16: *219624 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:53 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:54 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [error] 16#16: *218273 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:54 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:54 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:55 [error] 16#16: *215469 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:55 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:55 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
172.30.2.225 - - [17/Mar/2023:16:23:56 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:56 [error] 16#16: *219712 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:56 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "52.77.99.130"
172.30.2.225 - - [17/Mar/2023:16:23:56 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:56 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:56 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:57 [error] 18#18: *194837 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:57 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
172.30.2.225 - - [17/Mar/2023:16:23:58 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
172.30.2.225 - - [17/Mar/2023:16:23:58 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "46.114.35.116"
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
172.30.2.225 - - [17/Mar/2023:16:23:58 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "60.242.161.215"
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [warn] 17#17: 1024 worker_connections are not enough, reusing connections
172.30.2.225 - - [17/Mar/2023:16:23:58 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "52.77.99.130"
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
172.30.2.225 - - [17/Mar/2023:16:23:58 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
172.30.2.225 - - [17/Mar/2023:16:23:58 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "34.196.93.245"
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:58 [error] 18#18: *182395 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:58 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
172.30.2.225 - - [17/Mar/2023:16:23:59 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:23:59 [error] 16#16: *219760 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "POST //api/user/login HTTP/1.1", upstream: "http://10.152.183.179:8080/login", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:23:59 +0000] "POST //api/user/login HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:24:00 [warn] 16#16: 1024 worker_connections are not enough, reusing connections
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
172.30.2.225 - - [17/Mar/2023:16:24:00 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
172.30.2.225 - - [17/Mar/2023:16:24:00 +0000] "GET // HTTP/1.1" 200 2610 "-" "python-requests/2.26.0" "156.33.241.5"
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:00 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:01 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:01 [error] 18#18: *213973 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.2.225, server: localhost, request: "GET //api/user/uniqueid HTTP/1.1", upstream: "http://10.152.183.179:8080/uniqueid", host: "3.232.202.170:32512"
172.30.2.225 - - [17/Mar/2023:16:24:01 +0000] "GET //api/user/uniqueid HTTP/1.1" 502 494 "-" "python-requests/2.26.0" "98.142.103.241"
2023/03/17 16:24:01 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:01 [alert] 16#16: 1024 worker_connections are not enough
2023/03/17 16:24:01 [alert] 16#16: 1024 worker_connections are not enough
